src/resources/views/corporation/tracking.blade.php

```diff
@@ -6,23 +6,14 @@
 
 @section('corporation_content')
 
-  <div class="box box-default">
-    <div class="box-header with-border">
-      <h3 class="box-title">{{ trans('web::seat.tracking') }}</h3>
-    </div>
-    <div class="box-body">
-
-      <div>
-        <b>{{ trans('web::seat.status') }}</b>
-        <div class="input-group">
-          <label class="checkbox-inline">
-            <input onchange="filterme()" type="checkbox" name="token_status" value="valid_token" checked>{{ trans('web::seat.valid_token') }}
-          </label>
-          <label class="checkbox-inline">
-            <input onchange="filterme()" type="checkbox" name="token_status" value="invalid_token" checked>{{ trans('web::seat.invalid_token') }}
-          </label>
-        </div>
-      </div>
+  <div class="nav-tabs-custom">
+    <ul class="nav nav-tabs">
+      <li class="active"><a href="" data-toggle="tab" data-filter="all">{{ trans('web::seat.all') }} {{ trans('web::seat.tracking') }}</a></li>
+      <li><a href="" data-toggle="tab" data-filter="valid_token">{{ trans('web::seat.valid_token') }}</a></li>
+      <li><a href="" data-toggle="tab" data-filter="invalid_token">{{ trans('web::seat.invalid_token') }}</a></li>
+      <li><a href="" data-toggle="tab" data-filter="unseated">{{ trans('web::seat.none') }} {{ trans('web::seat.seat_user') }}</a></li>
```

Comment:
I'll prefer something like `Missing user` which is a bit less shorter and meaningful.